src/main/java/uk/gov/hmcts/reform/civil/enums/RespondentResponseTypeSpecPaidLess.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.civil.enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum RespondentResponseTypeSpecPaidLess {
```

Comment:
The class name suggests the type is always Paid Less. Can we rename it to something which conveys 'Paid status'

Reply:
Done